app-utils/plocate/spec

```diff
@@ -0,0 +1,5 @@
+VER=1.1.18
+REL=1
```

Comment:
Unlike Arch Linux, we start revision number from 0, which means you can omit this line when firstly introducing this package or updating.

```suggestion

```